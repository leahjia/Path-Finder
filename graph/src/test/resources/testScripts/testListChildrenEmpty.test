# DesignMap listing children nodes testing file.
# Tests that a list of destination nodes from a given node can be computed
# when there is no destination node

# Create a graph
CreateGraph map1

# Add one node to map1
AddNode map1 n1

# Finds the children of node n1
ListChildren map1 n1