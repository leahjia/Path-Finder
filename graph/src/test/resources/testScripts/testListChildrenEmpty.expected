# DesignMap listing children nodes testing file.
# Tests that a list of destination nodes from a given node can be computed
# when there is no destination node

# Create a graph
created graph map1

# Add one node to map1
added node n1 to map1

# Finds the children of node n1
the children of n1 in map1 are: